utils/bitfield/tests/bitfield_tests.rs

```diff
@@ -0,0 +1,186 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use bitfield::*;
+use bitvec::*;
+use fnv::FnvHashSet;
+use rand::{Rng, SeedableRng};
+use rand_xorshift::XorShiftRng;
+
+fn gen_random_index_set(range: u64, seed: u8) -> Vec<u64> {
+    let mut rng = XorShiftRng::from_seed([seed; 16]);
+
+    let mut ret = Vec::new();
+    for i in 0..range {
+        if rng.gen::<bool>() {
+            ret.push(i);
+        }
+    }
+    ret
```

Comment:
this could also be written with an [0..range].iter().filter(rng.gen::<bool>).map(|s|s).collect(). Just a suggestion. I usually prefer iterators over loops cause it is usually easier to read, don't have to disect what is happening in the loop

Reply:
great suggestion! I'll do that